heron/tools/common/src/python/utils/config.py

```diff
@@ -138,9 +139,22 @@ def get_heron_dir():
 
   :return: root location for heron-cli.
   """
-  path = "/".join(os.path.realpath(__file__).split('/')[:-9])
+  go_above_dirs = 9
+  path = "/".join(os.path.realpath(__file__).split('/')[:-go_above_dirs])
   return normalized_class_path(path)
 
+def get_zipped_heron_dir():
+  """
+  This will extract heron directory from .pex file,
+  with `zip_safe = False' Bazel flag added when building this .pex file
+
+  Specifically designed for heron-ui
```

Comment:
I am thinking any tool in the future that uses `zip_safe=False` can reuse this method, and naming this method in this way then makes sense.